src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -60,7 +60,8 @@ export class LocalGitRepoHelper {
     }
 
     public async getGitRemoteUrl(remoteName: string): Promise<string|void> {
-        return this.gitReference.remote(["get-url", remoteName]);
+        let remoteUrl = await this.gitReference.remote(["get-url", remoteName]);
+        return remoteUrl ? (<string>remoteUrl).trim() : remoteUrl;
```

Comment:
will trim() also trim trailing /

Reply:
It was removing whitespaces only. So have added the trailing logic to GitHub Helper which uses the remote url to extract repositoryId

Reply:
Also i was thinking of testing trailing '/' in case of azure Repo's. Will update if changes are required there as well

Reply:
As expected, it happened for azure repo's as well. Hence updated in the base repo.